pkg/app/pipedv1/plugin/toolregistry/toolregistrytest/toolregistrytest.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2024 The PipeCD Authors.
```

Comment:
Is this file a miss? 👀

Reply:
This package is a test helper for toolregistry.
The name `toolregistrytest` is like a [`slogtest`](https://pkg.go.dev/testing/slogtest@go1.23.2) in standard package.

Reply:
got your point, thanks 👍